src/frontend/components/form/isbnIsmnRegistrationForm/validation.js

```diff
@@ -129,7 +129,8 @@ export function validate(values) {
       {fieldName: 'zip', hasError: values.zip && !regexPatterns.zip.test(values.zip), message: 'postalAddress.zip.format'},
       {fieldName: 'city', hasError: values.city && !regexPatterns.city.test(values.city), message: 'format.city'},
       {fieldName: 'phone', hasError: values.phone && !regexPatterns.phone.test(values.phone), message: 'format.phone'},
-      {fieldName: 'publicationsPublic', hasError: !values.publicationsPublic || String(values.publicationsPublic).toLowerCase() !== 'true', message: 'publicationRegistrationIsbnIsmn.form.availability'},
```

Comment:
Proposing to still introduce the error but with empty error message. This allows displaying the user that the reasoning not allowing the continue to next step is due to error in this field validation.